examples/fitness/lib/date_utils.dart

```diff
@@ -13,6 +13,12 @@ class DateUtils {
   static const MS_IN_WEEK =
         DateTime.DAYS_PER_WEEK * Duration.MILLISECONDS_PER_DAY;
 
+  // TODO(jmesserly): locale specific date format
+  static String _twoDigits(int n) {
```

Comment:
https://pub.dartlang.org/packages/intl can help with this.  We're using it in the date picker:
https://github.com/eseidelGoogle/sky_date_picker
which collin should be landing in main line soon.